### PR TITLE
Student email length is limited to 45 chars #4518

### LINKED ISSUE #4518: Student email length is limited to 45 chars
This is a very short-sighted design decision. I don't know if there is an upper limit to email address length, but it has to allow more than 45 chars. We need to increase this immediately as one of our users is blocked by it.
Can do a hot fix quickly?

### PR DESCRIPTION
**Pull request by kanghj**
Fixes #4518

### CODE REVIEW

File: src/test/java/teammates/test/cases/ui/InstructorCourseStudentDetailsEditSaveActionTest.java [hunk 1]
```diff
@@ -133,7 +133,11 @@ public void testExecuteAndPostProcess() throws Exception{
         
         ______TS("Error case, invalid email parameter");
         
-        String invalidStudentEmail = "thisisaveryverylonglonglongstudentemailaccountname@gmail.tmt";
+        String invalidStudentEmail = "thisisaveryverylonglonglongveryverylonglonglong" 
+                                   + "veryverylonglonglongveryverylonglonglongveryvery"
+                                   + "longlonglongveryverylonglonglonglonglonglongveryverylonglonglong"
+                                   + "longlonglongveryverylonglonglonglonglonglongveryverlonglonglong"
+                                   + "studentemailaccountname@gmail.tmt";
```
Comment (damithc) [inline src/test/java/teammates/test/cases/ui/InstructorCourseStudentDetailsEditSaveActionTest.java]: I have a memory of a utility method that can generate a random string of a specific length. May be use that so that the length of the string is visible to the reader. Also, you can use it to test the boundary case (i.e. 255 chars).

Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/InstructorCourseStudentDetailsEditSaveActionTest.java]: Ok, will change to use that method. This string is actually a boundary case of 255 characters.

**Comment by kanghj** [comment]
Changes made. @LowWeiLin

**Comment by LowWeiLin** [comment]
Looks good, can merge.

**Comment by damithc** [comment]
Let's merge this soon. I'm hoping to do the release tonight.

**Comment by kanghj** [comment]
@damithc , merged V5.58.02 to release
